### ISSUE TITLE
Send it to Server returns ckeditor instance

### ISSUE THREAD

**Issue by scottshipman** [helios-ag/FMElfinderBundle#165]
I have the latest version of FMElfinderBundle (~5) and I am using the trsteel_ckeditor bundle as well. My configs are below:

When I hit Send it to the Server button, I get the popup dialog updated with what appears to be a ckeditor instance and the upload fails. See image at the bottom. I do not get any js console errors.

my web/static/uploads/media/ckeditor directory contains a .quarantine and a .tmb directory (not sure what these are, but they get added when I try to upload). But no files/images are uploaded. I am using the Image button and the Upload tab on that dialog.

in config.yml
```
trsteel_ckeditor:
    class: Trsteel\CkeditorBundle\Form\Type\CkeditorType
    transformers: ['tuihtmlpurifier'] #['html_purifier']
    toolbar: ['document', 'basicstyles', 'links', 'insert', 'styles', 'paragraph']
    toolbar_groups:
        document: ['Source', 'Undo', 'Redo']
        links: ['Link','Unlink','Anchor']
        insert: ['Embed', 'Image', 'Table']
        styles: ['Styles']
    ui_color: '#ffffff'
    startup_outline_blocks: false
    width: 100% #Integer or %
    height: 300 #Integer or %
    custom_config: "/assetic/ckeditorcustomsettings.js"
    filebrowser_upload_url:
            route: elfinder
            route_parameters:
                 instance: ckeditor
    filebrowser_image_browse_url:
            route: elfinder
            route_parameters:
                 instance: ckeditor
```
and also in config.yml
```
fm_elfinder:
    instances:
        ckeditor:
            locale:  en # defaults to current request locale
            editor: ckeditor # other options are tinymce, tinymce4, form, custom and simple
            fullscreen: true # defaults true, applies to simple and ckeditor editors
            theme: smoothness # jquery theme
            include_assets: true # disable if you want to handle loading of the javascript and css assets yourself
            connector:
                debug: false # defaults to false
                roots:       # at least one root must be defined
                    uploads:
                        show_hidden: false # defaults to false
                        driver: LocalFileSystem
                        path: static/uploads/media/ckeditor
                        upload_allow: ['image/png', 'image/jpg', 'image/jpeg']
                        upload_deny: ['all']
                        upload_max_size: 32M
```

I am using ckeditor v 4.35 (latest version of trsteel_ckeditor (1.8.4).
![screen shot 2015-10-01 at 5 35 31 pm](https://cloud.githubusercontent.com/assets/3016737/10234739/c552ac3c-6864-11e5-85eb-3cb89627bcc7.png)

**Comment by helios-ag** [comment]
Hi @scottshipman, elfinder is a filebrowser, so you need to work with first tab of insert image dialogue, switch to 'Image Info', and hit 'browse...' button.

**Comment by yellowmamba** [comment]
@helios-ag I have the same error. I also tried uploading to S3, according to https://github.com/helios-ag/FMElfinderBundle/blob/master/Resources/doc/flysystem.md, it does not work as well.

**Comment by helios-ag** [comment]
Hi @yellowmamba, well you can popup developer console inside your web browser, and look to response. Also take a look at issue #153

**Comment by helios-ag** [comment]
@yellowmamba i fixed issue with S3 v3 flysystem driver (PR #192/ related discussion in #191), you can test it with master branch.
